src/snowflake/snowpark/column.py

```diff
@@ -226,6 +228,7 @@ class Column:
     This class has methods for the most frequently used column transformations and operators. Module :mod:`snowflake.snowpark.functions` defines many functions to transform columns.
     """
 
+    # NOTE: For now assume Expression instances can be safely ignored when building AST
```

Comment:
can you add an example that's problematic here? Not sure what to do to resolve this note.

Reply:
This won't be an issue once all API functions that return `Column` instances also populate the `ast` parameter in the constructor.

For example currently because the `isin` method is not implemented to build the AST before returning, doing `df.filter(col("A").isin(1, 2, 3) & col("B"))` would fail since the boolean operator `&` would try to construct an AST using that of the new `col("A").isin(1, 2, 3)` column (which we currently don't fill if the only thing provided in the `Column` constructor is an `expr1` of type `Expression`.

Reply:
can we add the example?

Reply:
is there a TODO post phase 0?